### ISSUE TITLE
[Request] Sync Watched/Unwatched with Trakt

### ISSUE THREAD

**Issue by burekas7** [scakemyer/plugin.video.quasar#518]
Hi,

Can you add any support for sync with Trakt if items were watched on not?
Currently, just if I watched it through the addon it marks it with the Watched indicator tag.

Thanks.

**Comment by bluenote73** [comment]
I'm not clear what you are asking for here, can you be more specific?

**Comment by eidermar** [comment]
I think what he is asking is to sync the watched status of shows/movies with his Trakt account. Currently this is not the case. Only if you play an item via Quasar it will become tagged as watched within the plugin.

**Comment by burekas7** [comment]
Yes, currently I don't use quasar as a library in Kodi.
But most of the time I watch through quasar (using the Trakt's integration, Movies/TV shows collections etc)
The tag of 'Watched' status is shown on the item just if you play it through the addon, but if it was marked as 'watched' from other addon or through Trakt site, it doesn't show it when you in quasar addon.

Thanks.

**Comment by burekas7** [comment]
Hi scakemyer,
Is there any new update according to this issue?

Thanks.

**Comment by burekas7** [comment]
Trakt API:
* Sync  /   Get Watched  /   Get watched 
http://docs.trakt.apiary.io/#reference/sync/get-watched
(I think it's what relevant to this request)

**Comment by charly3pins** [comment]
@burekas7 I'm working on it in my forked repo. I'll link the PR to this issue when will be solved

**Comment by burekas7** [comment]
@charly3pins , Thanks man!

**Comment by charly3pins** [comment]
@burekas7 it's done! I added in the previous PR that I opened where also added the my shows and my next episodes categories. If @scakemyer don't merge to master, you can download from my repo and compile it. If you don't know how I can help you.

**Comment by burekas7** [comment]
@charly3pins, Thanks. I'm very appreciate your work.
I'll wait a little, if he will be delayed with it, I'll check it with your fork.

**Comment by burekas7** [comment]
@charly3pins 
Hi, how can I get a zip of your version that suits for my OS, for testing.
Thanks.

**Comment by charly3pins** [comment]
Hey @burekas7 , you can take the binary from my forked quasar-binaries repo https://github.com/charly3pins/quasar-binaries/tree/TraktIntegration
message me if any doubts or suggestions

**Comment by burekas7** [comment]
@charly3pins
Thanks. I have tried your version (Replace the exe file with yours, restart kodi, reactivate Trakt account), but it's still showing all my collection for example, it's not sync with their Trakt watched status, all of them still marked as "unwatched".

**Comment by charly3pins** [comment]
@burekas7 wich platform are you using? I tested in raspberry pi, mac os and windows 64 and worked successful.

**Comment by burekas7** [comment]
@charly3pins 
Win 64bit.
As I said, all the content in Quasar is marked as 'unwatched' unless I marked them manually as 'watched' in the past through the addon's context menu. There is no sync with their Trakt 'watched' status.

Do I need to refresh something or changing any setting or something else?

**Comment by charly3pins** [comment]
@burekas7 sorry man, I didn't understood you well before. The new feature that I added are the "My shows", where appears all your shows that you are watched every time and "My next episodes", where appears the next episode of each serie that you're currently watching. Also, I added the sync with Trakt when you see 80% or more of an episode or film, is marked as Watched in Trakt. 

The issue that I think are you saying to me is that the previous films/episodes that you had marked as watched in Trakt aren't marked as watched in Quasar, but this feature is not added, so I'm gonna unlink this issue from my PR because I see that is not what you wanted when you opened. Maybe in future I will try to add this.

Thanks for your feedback

**Comment by burekas7** [comment]
@charly3pins 
Yes, your changes are not belong to this issue, it relates to #580 that you already mentioned in your PR.
If you will be able to add a support about what I said in this issue it will be nice. This sync of the 'watched' status with Trakt, should be in all the categories in the addon.

And regarding your PR, I will mention you in #580 , because I have a comment about it. Thanks.
